docs/index.rst

```diff
@@ -41,7 +41,7 @@ See :doc:`community/devsetup` for setup instructions in case you want to develop
 
    tutorials/forecasting/quick_start_tutorial
    tutorials/forecasting/extended_tutorial
-   tutorials/data_manipulation/dataframesdataset
+   tutorials/data_manipulation/pandasdataframes
```

Comment:
This did not need to change (since the tutorial file name did not change)

Reply:
Wondering why the docs build did not fail?

Reply:
Because the file was actually renamed? ;)

Reply:
Ok, didn't see that